Synview-backend/index.ts

```diff
@@ -0,0 +1,36 @@
+import { Application, Router, oakCors } from "./deps.ts";
+import { router } from "./routes/user_routes.ts";
+import { Session } from "./deps.ts";
+type AppState = {
+  session: Session;
+};
+// import { parse } from "node:path";
+
+const Mainrouter = new Router();
```

Comment:
With a const like this, no need to upper-case, it's like the `app` and `env` variables just below - you'd just do `mainRouter`